### PR TITLE
#1016 Impediments.remove(...) + unit test and puzzle

### PR DESCRIPTION
**Pull request by amihaiemil**
PR for #1016 

* Registered the ``continue`` command inside ``q-tracker.xml``
* Added method ``Impediments.remove(...)``
* Added unit test for ``Impediments.remove(...)``
* Left puzzle for continuing implementation of the stakeholder.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#1378](https://github.com/zerocracy/farm/issues/1378) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1891340003); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/zerocracy/pm/in/ImpedimentsTest.java [hunk 1]
```diff
@@ -50,4 +50,31 @@ public void registerImpediment() throws Exception {
             Matchers.is(true)
         );
     }
+
+    @Test
+    public void removesImpediment() throws Exception {
```
Comment (krzyk) [inline src/test/java/com/zerocracy/pm/in/ImpedimentsTest.java]: @amihaiemil it would be good to have also a test for removal of not existing impediment

**Comment by krzyk** [comment]
@amihaiemil thanks

**Comment by krzyk** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1378](https://github.com/zerocracy/farm/issues/1378) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@krzyk According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by krzyk** [comment]
@ypshenychka I confirm

**Comment by ypshenychka** [comment]
@krzyk thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**